cppForSwig/cryptopp/Makefile

```diff
@@ -36,7 +37,6 @@ IS_SUN_CC = $(shell $(CXX) -V 2>&1 | $(EGREP) -c "CC: Sun")
 GAS210_OR_LATER = $(shell echo "" | $(AS) -v 2>&1 | $(EGREP) -c "GNU assembler version (2\.[1-9][0-9]|[3-9])")
```

Comment:
Afaik, this is based on the cryptopp Makefile from upstream. I thought it was easiest to leave as a regular Makefile, so I left it that way. Though I guess I could change it to use automake. I'm not sure if @droark or anyone else has an opinion on that. Maybe leaving it the way it is would make it easier to merge changes from upstream?